packages/typespec-test/test/versioning/generated/typespec-ts/src/index.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as v1 from "./v1";
+import * as v2 from "./v2";
```

Comment:
I wonder if we have considered use subpath export to support multiapi instead of using the sub namespaces?

Reply:
Yeah, we should definetly do that for ESM. Maybe this for CJS and sub exports for ESM. I'll handle ESM in a follow up PR